src/pymor/core/gather_inputs.py

```diff
@@ -277,6 +277,8 @@ def load_mfdataset(data, rule_spec):
     for f in all_files:
         logger.info(f"  * {f}")
     mf_ds = xr.open_mfdataset(all_files, parallel=True, use_cftime=True, engine=engine)
+    if "time_counter" in mf_ds.dims and "time" not in mf_ds.dims:
+        mf_ds = mf_ds.rename({"time_counter": "time"})
```

Comment:
I think a more general solution would be nice here. For example, you could add something to the rule like `time_dimname`:


```suggestion
    if rule.get("time_dimename") in mf_ds.dims and "time" not in mf_ds.dims:
        mf_ds = mf_ds.rename({rule.get("time_dimename"): "time"})
```